superset/viz.py

```diff
@@ -102,6 +103,8 @@
     "size",
 ]
 
+FILTER_VALUES_REGEX = re.compile(r"FILTER_VALUES\(['\"](\w+)['\"]\,", re.IGNORECASE)
```

Comment:
case-insensitive, allow `'` and `"`

Reply:
I think Jinja context variables/functions are case sensitive. Running `SELECT {{ FILTER_VALUES('region') }};` in SQL Lab gives me `undefined` errors.

Reply:
yes, you are right. filter_values is function not sql statement. I will fix this.